.github/workflows/docker.yml

```diff
@@ -59,4 +60,23 @@ jobs:
           # Docker compose can take some time to start
           max_attempts: 5
           timeout_minutes: 2
-          command: sleep 60 && linera --wallet docker/wallet.json --storage rocksdb:docker/linera.db sync-balance
\ No newline at end of file
+          command: sleep 60 && linera --wallet docker/wallet.json --storage rocksdb:docker/linera.db sync-balance
```

Comment:
(Not in this PR, but: This is already inside a retry loop, is it? Should we retry more often to avoid unnecessary delays, like every 10 seconds?)